superset/assets/javascripts/explore/components/ExploreActionButtons.jsx

```diff
@@ -17,9 +17,7 @@ export default function ExploreActionButtons({ canDownload, slice, query }) {
   return (
     <div className="btn-group results" role="group">
       <URLShortLinkButton slice={slice} />
-
```

Comment:
these extra lines make the code more readable IMO. seems like an unnecessary change in this PR.